zuul.d/jobs.yaml

```diff
@@ -494,3 +494,33 @@
     nodeset: vyos-1.1.8-python38
     vars:
       ansible_test_python: 3.8
+
+# VMware
+- job:
+    name: ansible-network-vmware-appliance
```

Comment:
we should call this

ansible-cloud-vmware-appliance, starting the 'cloud' jobs.